radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetworkController.java

```diff
@@ -120,8 +120,14 @@ private RadixNetworkController(
 			.map(epic -> epic.epic(connectableReducedNodeActions, networkState))
 			.collect(Collectors.toSet());
 
-		// FIXME: Cleanup disposable
-		Observable.merge(updates).subscribe(this::dispatch);
+		// FIXME: Cleanup disposable and error handling
```

Comment:
Is this `FIXME` still valid?

Reply:
Yes, should really not do a `subscribe` inside constructor but it has been good enough for now.